app/javascript/components/public_health/PatientsTable.js

```diff
@@ -93,6 +98,33 @@ class PatientsTable extends React.Component {
     // select tab and fetch patients
     this.handleTabSelect(tab);
 
+    // Select page if it exists in local storage
```

Comment:
Can we save the jurisdiction and assigned users filters as well?

Reply:
I couldn't get it working comfortably enough in time to meet the release schedule, so I'd like to do that as a followup.